operator/pkg/reconciliation/handler.go

```diff
@@ -159,8 +159,12 @@ func (rc *ReconciliationContext) isValid(dc *api.CassandraDatacenter) error {
 
 	// Basic validation up here
 
+	// validate the required superuser
 	errs = append(errs, rc.validateSuperuserSecret()...)
 
+	// validate any other defined users
+	errs = append(errs, rc.validateCassandraUserSecrets()...)
```

Comment:
I was thinking this might not be super fantastic for performance reasons, but then we load every pod resource for the `CassandraDatacenter` on every reconcile loop, so I guess it's fine.

Reply:
good thought - let's make a ticket to stress test this a little with 50 users

Reply:
KO-431